fe/fe-core/src/main/java/com/starrocks/metric/MetricRepo.java

```diff
@@ -704,6 +705,42 @@ public static void updateRoutineLoadProcessMetrics() {
         }
     }
 
+    private static void collectRoutineLoadIngestMetrics(MetricVisitor visitor, boolean minifyTableMetrics) {
+        List<RoutineLoadJob> jobs = GlobalStateMgr.getCurrentState().getRoutineLoadMgr()
+                .getRoutineLoadJobByState(
+                        Sets.newHashSet(RoutineLoadJob.JobState.RUNNING, RoutineLoadJob.JobState.PAUSED));
+
+        for (RoutineLoadJob job : jobs) {
+            try {
+                if (!minifyTableMetrics || job.getCommittedTaskNum() > 0) {
+                    CounterMetric<Long> metric =
+                            new LongCounterMetric("routine_load_committed_tasks", MetricUnit.NOUNIT,
+                                    "routine load committed tasks");
+                    addRoutineLoadLabel(metric, job);
+                    metric.increase(job.getCommittedTaskNum());
+                    visitor.visit(metric);
+                }
+
+                if (!minifyTableMetrics || job.getAbortedTaskNum() > 0) {
+                    CounterMetric<Long> metric =
+                            new LongCounterMetric("routine_load_aborted_tasks", MetricUnit.NOUNIT,
+                                    "routine load aborted tasks");
+                    addRoutineLoadLabel(metric, job);
+                    metric.increase(job.getAbortedTaskNum());
+                    visitor.visit(metric);
+                }
+            } catch (MetaNotFoundException ignored) {
+            }
+        }
+    }
+
+    private static void addRoutineLoadLabel(Metric<?> metric, RoutineLoadJob job) throws MetaNotFoundException {
+        metric.addLabel(new MetricLabel("job_name", job.getName()));
```

Comment:
is job level metric necessary?
is table level like `counterRoutineLoadBytesTotal` in `TableMetricsEntity` ok?

Reply:
done.
lable job_name removed.

Reply:
better to move the metrics into`TableMetricsEntity`

Reply:
Done moving the metrics `intoTableMetricsEntity`